multispaeti/_multispati_pca.py

```diff
@@ -35,9 +41,9 @@ class MultispatiPCA:
     ----------
     n_components : int or tuple[int, int], optional
         Number of components to keep.
-        If None, will keep all components (only supported for non-sparse `X`).
+        If None, will keep all components.
         If an int, it will keep the top `n_components`.
-        If a tuple, it will keep the top and bottom `n_components` respectively.
+        If a tuple, it will keep the top and bottom `n_components`, respectively.
```

Comment:
maybe somewhere in the documentation you need to explain what bottom and top components mean or cite the paper where it is explained

Reply:
The paper(s) is linked in the docs and in the class constructor